usaspending_api/download/management/commands/download_sqs_worker.py

```diff
@@ -87,6 +88,11 @@ def download_service_app(download_job_id):
         span_type=SpanTypes.WORKER,
     ) as span:
         download_job = _retrieve_download_job_from_db(download_job_id)
+        limit = download_job.json_request.get("limit")
+        if limit is not None and limit > settings.MAX_DOWNLOAD_SIZE:
+            raise Exception(
+                f"Unable to process this download because it includes more than the current limit of {settings.MAX_DOWNLOAD_SIZE} records"
+            )
```

Comment:
I think you should put this after the first log message and after the span.set_tags (i.e. between
```
        span.set_tags(download_job_details)
        generate_download(download_job=download_job)
```

That way we have some traceability in the log and in the datadog trace of this download request coming in and attempting to start it.